src/auth/dto/authcreds.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { IsString, IsNotEmpty, IsEmail } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class AuthCredsDto {
+  @IsString()
+  @ApiProperty({ description: 'The email of the user' })
+  @IsEmail()
+  @IsNotEmpty()
+  email: string;
+
+  @IsString()
+  @ApiProperty({ description: 'The password of the user' })
+  @IsNotEmpty()
+  password: string;
```

Comment:
PullProofAI

The password field seems to be correctly validated with `@IsString()` and `@IsNotEmpty()`. No issues found here.


<!-- This is an auto-generated comment by PullProofAI -->